chsdi/templates/htmlpopup/base.mako

```diff
@@ -53,7 +53,7 @@
 <body>
   <div class="chsdi-htmlpopup-container">
 % else:
-  <div class="htmlpopup-container">
+  <div id="${c['layerBodId']}#${c['featureId']}" class="htmlpopup-container">
```

Comment:
I'm quite sure that we did encounter issues. But maybe it's solved now.

2015-09-17 15:21 GMT+02:00 Gilbert Jeiziner <notifications@github.com>:

> In chsdi/templates/htmlpopup/base.mako
> <https://github.com/geoadmin/mf-chsdi3/pull/1660#discussion_r39743608>:
>
> > @@ -53,7 +53,7 @@
> >  <body>
> >    <div class="chsdi-htmlpopup-container">
> >  % else:
> > -  <div class="htmlpopup-container">
> > +  <div id="${c['layerBodId']}#${c['featureId']}" class="htmlpopup-container">
>
> yes. Otherwise, we would not be capable to get the geometry for a feature.
> We construct the url using layerBodId and featureId. Never faced a problem
> because of that, AFAIK.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geoadmin/mf-chsdi3/pull/1660/files#r39743608>.
>



-- 
Twitter: http://twitter.com/cedricmoullet
Linked In: http://www.linkedin.com/in/cedricmoullet

<http://twitter.com/cedricmoullet>